source/docs/articles/sites/code/using-the-pantheon-workflow.md

```diff
@@ -63,6 +63,9 @@ Of course, manually making configuration changes as part of your deployment is a
 
 Ultimately, the right answer for managing configuration updates to your site depends on your use case and comfort level with these techniques. It's an ongoing debate in the Drupal and WordPress universe, with new solutions being developed all the time.
 
+### See also

```

Comment:
@ari-gold Usually the See Also is at the end of the article. If it's needed in this section, I can revise it to add it to the previous paragraph. Please lmk!